android/settings_aar.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
I do not remember explicitly changing this file. Right now it has some symbol not found errors however I read that Flutter does not use Gradle so we may not even need these files.

Reply:
So it seems to crash because in build.gradle there is a dependency     project.evaluationDependsOn(':app') on line 25. I never made these changes so I can assume there were also auto generated.